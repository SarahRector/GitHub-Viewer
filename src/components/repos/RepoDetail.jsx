import React from 'react';
import PropTypes from 'prop-types';

const RepoDetail = ({ name, url }) => (
  <div>
    <h1>{name}</h1>
    <p>{url}</p>
  </div>
);

RepoDetail.propTypes = {
  name: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired
};

export default RepoDetail;
