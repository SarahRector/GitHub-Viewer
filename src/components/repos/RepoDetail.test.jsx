import React from 'react';
import { render, cleanup } from '@testing-library/react';
import RepoDetail from './RepoDetail';

describe('App component', () => {
  afterEach(() => cleanup());
  it('renders RepoDetail', () => {
    const { asFragment } = render(<RepoDetail
      name="test repo"
      url="test.com" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
