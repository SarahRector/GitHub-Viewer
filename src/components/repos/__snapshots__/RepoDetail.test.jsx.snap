// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders RepoDetail 1`] = `
<DocumentFragment>
  <div>
    <h1>
      test repo
    </h1>
    <p>
      test.com
    </p>
  </div>
</DocumentFragment>
`;
